dist/bin/scaladoc

```diff
@@ -59,6 +59,7 @@ classpathArgs () {
   CLASS_PATH+="$(find_lib "*scala3-interfaces*")$PSEP"
   CLASS_PATH+="$(find_lib "*scala3-library*")$PSEP"
   CLASS_PATH+="$(find_lib "*tasty-core*")$PSEP"
+  CLASS_PATH+="$(find_lib "*scala3-staging*")$PSEP"
```

Comment:
It seems to me that the CI failure re: `inkuire-db.json` is caused by this addition.

Reply:
@griggt Thanks for your input. Removing the `scala3-staging` entry did indeed solve the issue.